components/gantry/multiaxis/multiaxis.go

```diff
@@ -229,6 +229,21 @@ func (g *multiAxis) IsMoving(ctx context.Context) (bool, error) {
 	return g.opMgr.OpRunning(), nil
 }
 
+func (g *multiAxis) Kinematics(ctx context.Context) (referenceframe.Model, error) {
+	if g.model == nil {
+		model := referenceframe.NewSimpleModel("")
+		for _, subAx := range g.subAxes {
+			k, err := subAx.Kinematics(ctx)
```

Comment:
This will change the behavior of multiaxis gantries as previously `ModelFrame` would produce a model and now it won't ass all normal gantries return `nil, unimplementeErr` from `Kinematics()`

Is that desired?

Reply:
Actually this may go over the network boundary I will have to verify.

Reply:
Ok never mind this is not a concern.  This is working for the same reason that `ModelFrame` worked before we just renamed it.  Gantry doesn't even have Kinematics on its API